JSTests/test262/expectations.yaml

```diff
@@ -654,6 +654,24 @@ test/built-ins/AsyncGeneratorPrototype/return/return-suspendedStart-broken-promi
 test/built-ins/AsyncGeneratorPrototype/return/return-suspendedYield-broken-promise-try-catch.js:
   default: 'Test262:AsyncTestFailure:Error: broken promise'
   strict mode: 'Test262:AsyncTestFailure:Error: broken promise'
+test/built-ins/Date/S15.9.3.1_A5_T1.js:
+  default: 'Test262Error: Expected Fri Dec 01 1899 00:00:00 GMT+0009 (Central European Standard Time) to be -2211667200000 milliseconds from the Unix epoch'
+  strict mode: 'Test262Error: Expected Fri Dec 01 1899 00:00:00 GMT+0009 (Central European Standard Time) to be -2211667200000 milliseconds from the Unix epoch'
+test/built-ins/Date/S15.9.3.1_A5_T2.js:
+  default: 'Test262Error: Expected Sun Dec 31 1899 00:00:00 GMT+0009 (Central European Standard Time) to be -2209075200000 milliseconds from the Unix epoch'
+  strict mode: 'Test262Error: Expected Sun Dec 31 1899 00:00:00 GMT+0009 (Central European Standard Time) to be -2209075200000 milliseconds from the Unix epoch'
+test/built-ins/Date/S15.9.3.1_A5_T3.js:
+  default: 'Test262Error: Expected Sun Dec 31 1899 23:00:00 GMT+0009 (Central European Standard Time) to be -2208992400000 milliseconds from the Unix epoch'
+  strict mode: 'Test262Error: Expected Sun Dec 31 1899 23:00:00 GMT+0009 (Central European Standard Time) to be -2208992400000 milliseconds from the Unix epoch'
+test/built-ins/Date/S15.9.3.1_A5_T4.js:
+  default: 'Test262Error: Expected Sun Dec 31 1899 23:59:00 GMT+0009 (Central European Standard Time) to be -2208988860000 milliseconds from the Unix epoch'
+  strict mode: 'Test262Error: Expected Sun Dec 31 1899 23:59:00 GMT+0009 (Central European Standard Time) to be -2208988860000 milliseconds from the Unix epoch'
+test/built-ins/Date/S15.9.3.1_A5_T5.js:
+  default: 'Test262Error: Expected Sun Dec 31 1899 23:59:59 GMT+0009 (Central European Standard Time) to be -2208988801000 milliseconds from the Unix epoch'
+  strict mode: 'Test262Error: Expected Sun Dec 31 1899 23:59:59 GMT+0009 (Central European Standard Time) to be -2208988801000 milliseconds from the Unix epoch'
+test/built-ins/Date/S15.9.3.1_A5_T6.js:
+  default: 'Test262Error: Expected Sun Dec 31 1899 23:59:59 GMT+0009 (Central European Standard Time) to be -2208988800001 milliseconds from the Unix epoch'
+  strict mode: 'Test262Error: Expected Sun Dec 31 1899 23:59:59 GMT+0009 (Central European Standard Time) to be -2208988800001 milliseconds from the Unix epoch'
```

Comment:
I don't get how those changes introduce this -- Feels like it's something locally wrong on my machine but I'll follow-up on this tomorrow.

Reply:
So I think I should probably not introduce this diff here: I got those failure on my machine even from main branch (as well as a bunch of timedate related tests running form run-javascriptcore-tests). Let's remove that?